apps/emqx_resource/src/emqx_resource_manager.erl

```diff
@@ -865,56 +994,6 @@ get_channel_status_channel_added(#data{id = ResId, mod = Mod, state = State}, Ch
     RawStatus = emqx_resource:call_channel_health_check(ResId, ChannelId, Mod, State),
```

Comment:
Can the manual channel health check block the resource manager? Can this cause any problems?

Reply:
https://github.com/emqx/emqx/pull/12830